fern/ir-types-v29/definition/types.yml

```diff
@@ -288,3 +293,8 @@ types:
     properties:
       typeName: DeclaredTypeName
       shape: ExampleTypeShape
+
+  ExampleNamedObjectType:
+    properties:
+      typeName: DeclaredTypeName
```

Comment:
`typeId`

Reply:
We can actually just use `ExampleObjectType` directly.